src/firefly/js/tables/ui/TableRenderer.js

```diff
@@ -74,7 +76,7 @@ export const HeaderCell = React.memo( ({col, showUnits, showTypes, showFilters,
     const {label, name, desc, sortByCols, sortable} = col || {};
     const cdesc = desc || label || name;
     const sortDir = SortInfo.parse(sortInfo).getDirection(name);
-    const sortCol = sortByCols || name;
+    const sortCol = sortByCols ? splitCols(sortByCols) : [name];
```

Comment:
sortCol is now an array of strings instead of string, but onSort's signature hasn't changed - not sure why is it still working.

Reply:
That's because onSort is only passing the param along to `SortInfo.toggle(cnames=[])`.